verilog/analysis/checkers/no_trailing_spaces_rule_test.cc

```diff
@@ -62,6 +63,7 @@ TEST(NoTrailingSpacesRuleTest, RejectsTrailingSpaces) {
       {"a  b", {kToken, "   "}, "\n\n1  2", {kToken, "  "}, "\n"},
       {"module foo;", {kToken, " "}, "\nendmodule\n"},
       {"module foo;\nendmodule", {kToken, " "}, "\n"},
+      {"module foo;\nendmodule", {kToken, "  "}, "\r\n"},
```

Comment:
Similar here. We essentially need to check that the newline tokenization works as it should (which happens somewhere in the lexer, but we need to check that it doesn't get confused with `\r\n`. We expect not, but adding a test-case will help us make sure.
`{"module foo;\nendmodule", {kToken, "  "}, "\r\n\r\n"},`

Reply:
I've added this to the test, thank you for suggesting the cases.